### ISSUE TITLE
Is halite a dependency or a dev_dependency?

### ISSUE THREAD

**Issue by maxlinc** [poise/poise#24]
I'm currently working in a pretty locked down organization, so the fewer gems I need to ask to be allowed the simpler it is to get approved.

I noticed that poise only has one dependency, halite, and then halite has several more. However, it looks like halite is just used for developing and releasing gems (including poise) as a cookbook and isn't actually part of implementing the poise functionality itself. Could halite be changed from a dependency to a developement_dependency (like poise-boiler)?

**Comment by coderanger** [comment]
No, having halite as a runtime dependency is what marks a cookbook as a "halite-style cookbook gem". That said, you don't need it your environment, it's all baked to actual Chef cookbooks before upload to Supermarket so when you consume it from there you don't need to know anything about my weirdo development style. You would not generally consume Poise and my other libs as actual gems.
